### PR TITLE
product switcher header to fiori3 concept

### PR DESCRIPTION
**Pull request by JohannesDoberer**

### CODE REVIEW

File: core/examples/luigi-sample-angular/e2e/tests/context-switcher.spec.js [hunk 1]
```diff
@@ -96,6 +96,8 @@ describe('Context switcher', () => {
       'Remove Project'
     );
 
+    cy.wait(3000);
```
Comment (marynaKhromova) [inline core/examples/luigi-sample-angular/e2e/tests/context-switcher.spec.js]: why do we need it here?

Comment (JohannesDoberer) [reply core/examples/luigi-sample-angular/e2e/tests/context-switcher.spec.js]: fixed failing e2e test, but it seems it is not necessary anymore.

File: core/src/navigation/ProductSwitcher.html [hunk 2]
```diff
@@ -23,19 +23,26 @@
...
             <ul class="fd-menu__list">
-              {#each productSwitcherItems as productSwitcherItem} {#if
-              productSwitcherItem.icon && productSwitcherItem.label}
+              {#each productSwitcherItems as productSwitcherItem}
+              {#if productSwitcherItem.label}
               <li on:click="onActionClick(productSwitcherItem)">
                 <a class="fd-menu__item">
-                  <span class="fd-product-switcher__product-icon">
-                    <img src="{productSwitcherItem.icon}">
+                  <span
```
[the remark this answers is not in the document]
Comment (jesusreal) [reply core/src/navigation/ProductSwitcher.html]: Discussed, will be done in scope of a refactoring task later, as it is more job than I thought

File: core/src/navigation/ProductSwitcher.html [hunk 3]
```diff
@@ -52,42 +59,43 @@
...
               class="y-full-width-list__item"
             >
               <span
-                class="y-full-width-list__icon"
-                style="background-image: url({productSwitcherItem.icon})"
-              ></span>
-              <span class="y-full-width-list__title">{productSwitcherItem.label}</span>
-              <span class="y-full-width-list__subtitle">
-                <!-- Here comes the subtitle which is unknown for now -->
+                class="y-full-width-list__icon {productSwitcherItem.icon && hasOpenUIicon(productSwitcherItem) ? 'sap-icon--' + productSwitcherItem.icon + ' sap-icon--m' : '' }"
```
[the remark this answers is not in the document]
Comment (jesusreal) [reply core/src/navigation/ProductSwitcher.html]: Discussed, will be done in scope of a refactoring task later, as it is more job than I thought

File: core/src/navigation/ProductSwitcher.html [hunk 4]
```diff
@@ -52,42 +59,43 @@
...
+      <div class="fd-action-bar">
+        <div class="fd-action-bar__back">
+          <button
+            class="fd-button--light fd-button--compact sap-icon--nav-back"
+            on:click="toggleDropdownState(event)"
+            data-cy="mobile-product-switcher-close"
+          ></button>
+        </div>
+        <div class="fd-action-bar__header">
+          <h5 class="fd-action-bar__title fd-has-type-1">My Product</h5>
```
[the remark this answers is not in the document]
Comment (jesusreal) [reply core/src/navigation/ProductSwitcher.html]: Done

**Pull request merged**